contracts/contracts/NoLossDao_v0.sol

```diff
@@ -338,55 +383,49 @@ contract NoLossDao_v0 is Initializable {
 
   /// @dev Anyone can call this every 2 weeks (more specifically every *iteration interval*) to receive a reward, and increment onto the next iteration of voting
   function distributeFunds() external iterationElapsed {
-    // On a *whatever we decide basis* the funds are distributed to the winning project
-    // E.g. every 2 weeks, the project with the most votes gets the generated interest.
-    // figure our what happens with the interest from the first proposal iteration
-    // Possibly make first iteration an extended one for our launch (for marketing)
-    // console.log(
-    //   'Iteration no: ',
-    //   proposalIteration,
-    //   'Time that this iteration has ended incremented ',
-    //   proposalDeadline
-    // );
+    interestReceivers[0] = msg.sender; // Set the miners address to receive a small reward
```

Comment:
I don't have a better way to do it in mind, but it should definitely be clearly documented that index 0 is always the miner reward.